python-package/lightgbm/sklearn.py

```diff
@@ -992,7 +993,7 @@ def fit(self, X, y,
                    + _base_doc[_base_doc.find('eval_init_score :'):])  # type: ignore
     _base_doc = fit.__doc__
     _before_early_stop, _early_stop, _after_early_stop = _base_doc.partition('early_stopping_rounds :')
-    fit.__doc__ = (_before_early_stop
-                   + 'eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n'
-                   + ' ' * 12 + 'The evaluation positions of the specified metric.\n'
-                   + ' ' * 8 + _early_stop + _after_early_stop)
+    fit.__doc__ = (f"{_before_early_stop}"
+                   f" eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n"
+                   f"{' ':12} The evaluation positions of the specified metric.\n"
+                   f"{' ':8} {_early_stop} {_after_early_stop}")
```

Comment:
```suggestion
    fit.__doc__ = (_before_early_stop
                   "eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n"
                   f"{' ':12}The evaluation positions of the specified metric.\n"
                   f"{' ':8}{_early_stop}{_after_early_stop}")
```

I think these changes have introduced some extra spaces. It's also not necessary to add an f-string around a value that is a single variable, e.g. `_before_early_stop` does not need to be changed to `f"{_before_early_stop}"`. In that case, the templating with f-strings isn't changing anything about the string.